omnibus/cookbooks/omnibus-supermarket/attributes/default.rb

```diff
@@ -168,10 +168,10 @@
 default['supermarket']['nginx']['default']['modules'] = []
 
 # ## Postgres
-
+default['supermarket']['postgresql']['version'] = '13.3'
 default['supermarket']['postgresql']['enable'] = true
 default['supermarket']['postgresql']['username'] = node['supermarket']['user']
-default['supermarket']['postgresql']['data_directory'] = "#{node['supermarket']['var_directory']}/postgresql/13.3/data"
+default['supermarket']['postgresql']['data_directory'] = "#{node['supermarket']['var_directory']}/postgresql/#{node['supermarket']['postgresql']['version']}/data"
```

Comment:
Depending on how this attribute gets set this isn't going to work the way you think. It can potentially be done lazily if we upgrade chef client in supermarket-ctl. Paging @lamont-granquist